### PR TITLE
Add visual feedback while creating 2fa backup codes

### LINKED ISSUE #19564: No visual feedback while backup codes are generated
### Steps to reproduce
1. Run Nextcloud on a slow / embedded device.
2. Generate backup codes for the two-factor authentication.

### Expected behaviour
A spinner / some visual feedback should be shown while the codes are generated on the server.

### Actual behaviour
No feedback is visible until some time later the codes are being shown. This might cause users to think the generation did not work so they click the button again (causing even more load on the device).

### Server configuration

This is an ODROID C1 running Nextcloud 18.0.0 on Ubuntu 18.04 with php-fpm 7.2.24 and MariaDB 10.1.44 from the official repository.

### PR DESCRIPTION
**Pull request by joeried**
This should resolve #19564

Adds loading icon and disables the button while backup code creation is in progress.

@fancycode @kesselb

### CODE REVIEW

File: apps/twofactor_backupcodes/src/views/PersonalSettings.vue [hunk 1]
```diff
@@ -2,7 +2,9 @@
 	<div>
 		<button v-if="!enabled"
 			id="generate-backup-codes"
+			:disabled="generatingCodes"
 			@click="generateBackupCodes">
+			<span :class="{'icon-loading-small': generatingCodes}" />
```
Comment (kesselb) [inline apps/twofactor_backupcodes/src/views/PersonalSettings.vue]: Please move the span behind text.

**Review by kesselb — APPROVED** [review]
Thanks :+1:

File: apps/twofactor_backupcodes/src/views/PersonalSettings.vue (hunk 1 above)
Comment (kesselb) [reply apps/twofactor_backupcodes/src/views/PersonalSettings.vue]: And if you made those changes you can use our build bot for the node stuff. Just add a new comment to your pull request: "/compile amend /"

**Review by ChristophWurst — APPROVED** [review]
Tested and works very nicely! Thanks a lot!

**Comment by ChristophWurst** [comment]
Please rebase & rebuild the branch. Ignore merge conflicts in the build artefacts. Let us know if you need any help!

**Pull request merged**